xbmc/cores/VideoPlayer/VideoRenderers/windows/RendererBase.cpp

```diff
@@ -653,7 +653,7 @@ void CRendererBase::ProcessHDR(CRenderBuffer* rb)
     {
       // Switch to SDR rendering
       CLog::LogF(LOGINFO, "Switching to SDR rendering");
-      // not need set color space because is alredy set when swap chain is re-created
+      // color space already sdr or set by the swap chain re-creation
       if (m_AutoSwitchHDR)
       {
         if (DX::Windowing()->IsHDROutput())
```

Comment:
ok for the text I was suggesting moving it where more appropriate
```suggestion
      if (m_AutoSwitchHDR)
      {
        // color space already sdr or set by the swap chain re-creation
        if (DX::Windowing()->IsHDROutput())
```